pubspec.yaml

```
@@ -1,6 +1,6 @@
 name: fluent_ui
 description: Implements Windows UI in Flutter. Based on the official documentation
-version: 4.5.0
+version: 4.5.1
```

```suggestion
version: 4.5.0
```

this should be maintained as it is

---

Ok sorry i solved it